data/colorTest/index.html

```diff
@@ -0,0 +1,10 @@
+<html>

```

Comment:
I think these two files are starting to abuse the notion of "data". To me, if the code in these files will be executed, then it is not data. The other files are data in my mind because we're not ever treating them as code to be executed

I'm not sure where to put them, though. Maybe we should just remove them?

Reply:
I think the playground files are really useful as well as this new one. I agreed that a term data is not applicable